lib/streamlit/elements/arrow_vega_lite.py

```diff
@@ -203,8 +202,8 @@ def marshall(
     "x2",
     "y2",
     "xError",
-    "yError2",
```

Comment:
drive-by change: these keys where duplicated and I assume it was actually supposed to be added as:

```
"xError2",
"yError",
```